resources/logical_interconnect_group.rb

```diff
@@ -38,15 +40,57 @@ def load_lig(item)
         raise "#{up[:networkType]} #{network_name} not found" unless net.retrieve!
         up.add_network(net)
       end
-      uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
+      parsed_uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
       item.add_uplink_set(up)
     end
     item
   end
+
+  # like? function specific to the interconnectMapTemplate resource
+  def interconnect_map_like?(item, another_item)
```

Comment:
Yeah, I think we could definitely come up with something good in the SDK and just let the Chef cookbook struggle with the updates part of it.
In the SDK, we can just override the like? or recursive_like? method to handle that specific resource correctly (the same way we do create, etc.).